### ISSUE TITLE
How to used it programatically

### ISSUE THREAD

**Issue by Kikobeats** [lewiscowper/markdown-chalk#27]
```
var markdownChalk = require('markdown-chalk');

var result = markdownChalk('# Some Markdown string');

console.log(result);
```

this doesn't work because `index.js` doesn't exist for any reason ¯\_(ツ)_/¯

**Comment by lewiscowper** [comment]
So I've closed this issue as it now works, but unfortunately this package is ideally useful for people to use it as a CLI tool, so there's lots of spacing and default that work for my use cases in the terminal.

I have fixed it so you could use it as you describe, and published version 2.1.0 to npm along with it.

However if you want to use it programmatically and disagree with my choices regarding colours or newlines etc, feel free to fork it and use it as a base for your own custom [marked](https://www.npmjs.com/package/marked) renderer. :)

**Comment by Kikobeats** [comment]
no problem, thanks!
